core/templates/services/audio-bar-status.service.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to notify about status of AudioBar
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AudioBarStatusService {
+    audioBarIsExpanded: boolean = false;
```

Comment:
TS files should have 2-space indentation.

@Hudda shouldn't this be part of the linter checks? If not, we should add it to #8423...?

Reply:
@seanlip Done. Added to #8423

Reply:
Thanks @Hudda!

Just one follow-up question. This feels like a pretty standard built-in sort of check. Shouldn't it be trivial to add with eslint or something? (I'm not sure, but just thought I'd ask, since I've been coming across some of these checks which "feel like they should be implemented already but aren't".)

Reply:
Ok Fixed.

Reply:
@seanlip This check is not yet supported by eslint, see comment [here](https://github.com/eslint/eslint/issues/11719#issuecomment-493033691).

Reply:
Ah, interesting. OK, thanks for pointing me to it!